backends/openvino/README.md

```diff
@@ -0,0 +1,88 @@
+# OpenVINO Backend for ExecuTorch
+The OpenVINO backend enables optimized execution of deep learning models on Intel hardware, leveraging Intel's [OpenVINO toolkit](https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/overview.html) for inference acceleration.
+
+## Supported Hardware
```

Comment:
Might be worth listing examples of such hardwrae available in the market

Reply:
Updated readme file to include a link for supported hardware.